config.cmake

```diff
@@ -460,4 +460,13 @@ config_string(
     DEPENDS "KernelIsMCS" UNDEF_DISABLED
 )
 
+config_string(
+    KernelStaticMaxBudgetUs KERNEL_STATIC_MAX_BUDGET_US
+    "Specifies a static maximum to which scheduling context can have \
+    either its period or budget configured."
+    DEFAULT 0
```

Comment:
0 doesn't seem like a sensible default?

Reply:
I believe the intention is to use `getMaxUsToTicks()` if this is 0.  You were the author though, so you may know your own intention better than me

Reply:
Lol, I do not remember :)

Reply:
This only gets set to a non-zero value (a week) for the verified configuration and 0 otherwise (in which case maxUsToTicks) is used.